server/lomas_server/models/responses.py

```diff
@@ -0,0 +1 @@
+# TODO issue
```

Comment:
is there an issue ? can you add the ticket number

Reply:
Yes, issue 336, added it.